README.md

```diff
@@ -222,7 +222,7 @@ bin/console upgradability:php:upgrade
 
 ## Enable project change integration
 ​
-* Turn on integrator trigger after package update step
+* Added config for enabling integrator (default disabled)
```

Comment:
I was referring to PR title :)
The text here was fine.

Reply:
Thanks, I got it
* Readme test was reverted 
* PR title was changes